client/src/components/App.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { FaBeer } from 'react-icons/fa';
+import { FaHeart } from 'react-icons/fa';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.handleClickTest = this.handleClickTest.bind(this);
+  }
+
+  handleClickTest() {
```

Comment:
Same comment as before - Name your event handler in the following format:

handle[thing we're interacting with][how we're interacting with it]

Example:

`handleSubmitButtonClick`

Reply:
This might mean making more than one method, one for each button!